dadospb/utils/TemplateDownload.py

```diff
@@ -31,6 +34,7 @@ class TemplateDownload():
             output_dir (str): diretôrio de saída
             periodic_data (bool): se os dados são atualizados periodicamente
     '''
+    PANDAS_READ_METHODS = {"csv": pd.read_csv, "json": pd.read_json}
```

Comment:
Uma solução para o comentário da linha 380.

```suggestion
    PANDAS_READ_METHODS = {"csv": lambda data: pd.read_csv(data, sep=';'), "json": pd.read_json}
```

Reply:
Eu não entendi o porquê do `sep=';'` resolver o problema de _retornar o dados pré-processados para alteração dos nomes das colunas e possíveis correções dos dados_ (comentário da linha 380).

Pode explicar melhor o que esse `lambda` envelopando o argumento `sep` resolve? Aí para eu entender q q tá pegando e consigo opinar hehehe...

Reply:
Pq quero pensar melhor essa solução? Se a solução for passar `sep=';'` mesmo, acho que vale extrair para um método ao invés de [usar e abusar do `lambda`](https://treyhunner.com/2018/09/stop-writing-lambda-expressions/). Como estamos atribuindo uma função a um identificador, a [PEP8 pede que a gente não use `lambda`](https://www.python.org/dev/peps/pep-0008/#id51) -- mas `functools.partial` pode ser uma solução mais elegante.

Mas, como eu disse, não entendi pq o `sep=';'` se relaciona com o comentário da _linha 380_ ainda, então fica difícil dar pitaco.

Reply:
O `sep=';'` é só pra a leitura dos dados mesmo. O processamento dos dados é algo a parte pra padronizar o nome das colunas e alguns ajustes dos dados caso possível.

Reply:
Realmente, com o `functools.partial` fica melhor